### PR TITLE
add citation info. for hll algorithm 

### PR DESCRIPTION
**Pull request by alameldin**
fixes issue #824

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, add to testlist

**Comment by alameldin** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2015-02-23  Hussien Alameldin  <hussien@msu.edu>
+
+   * khmer/khmer_args.py: add 'hll' citation entry "Irber and Brown, unpublished." to  _alg. dict.
+   * sandbox/unique-kmers.py: add call to 'info' with 'hll' in the algorithms list.
```
Comment (mr-c) [inline ChangeLog]: Lines need wrapping at 80 characters. In this case 'unpublished ...' and 'list.' need to go on the next line.

**Comment by mr-c** [comment]
Other than the formatting fix this is ready for merging

**Comment by mr-c** [comment]
@alameldin Thanks! Alas there are merge conflicts; can you bring the pull request up to date with the master branch?

**Comment by ctb** [comment]
Superseded by #845.

**Pull request closed without merging**